bucket/forkgram.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "4.0.0",
+    "description": "Fork of Telegram Desktop messaging app.",
+    "homepage": "http://t.me/forkgram",
+    "license": {
+        "identifier": "GPL-3.0-with-OpenSSL-exception",
+        "url": "https://github.com/Forkgram/tdesktop/blob/dev/LICENSE"
+    },
+    "url": "https://github.com/Forkgram/tdesktop/releases/download/v4.0.0/Telegram.zip",
+    "hash": "sha1:e2a57ee049afb3d9dc78572c68c05a7a4cd8b205 ",
```

Comment:
We use **SHA-256** by default because it is more reliable.

Please use SHA-256 for `hash`, unless the developer of the app provide hash files other than SHA-256.
Also, there is a trailing space in the hash., which is causing CI to fail.